lib/utils/under-dash.js

```diff
@@ -84,12 +84,39 @@ const _ = {
   },
 
   escapeHtml(html) {
-    return html
-      .replace(/&/g, '&amp;')
-      .replace(/</g, '&lt;')
-      .replace(/>/g, '&gt;')
-      .replace(/"/g, '&quot;')
-      .replace(/'/g, '&#039;');
+    const regexResult = escapeHtmlRegex.exec(html);
+    if (!regexResult) return html;
+
+    let result = '';
+    let escape = '';
+    let lastIndex = 0;
+    let i = regexResult.index;
+    for (; i < html.length; i++) {
+      switch (html.charCodeAt(i)) {
```

Comment:
For readability I would use html.charAt() and the actual characters in case ('>' vs 62). I tested both variants and performance is identical.

Reply:
Good catch!